reachy_sdk/reachy_sdk.py

```diff
@@ -50,17 +54,33 @@ class ReachySDK:
     - all force sensors (can be accessed directly via their name or via the force_sensors list).
     - all fans (can be accessed directly via their name or via the fans list).
 
-    The synchronisation with the robot is automatically launched at instanciation and is handled in background automatically.
+    The synchronisation with the robot is automatically launched at instanciation and is handled in background
+    automatically.
     """
 
-    def __init__(self, host: str, sdk_port: int = 50055, camera_port: int = 50057, restart_port: int = 50059) -> None:
+    def __init__(
+                self,
+                host: str,
+                with_mobile_base: bool = False,
+                sdk_port: int = 50055,
+                camera_port: int = 50057,
+                restart_port: int = 50059
+                ) -> None:
         """Set up the connection with the robot."""
+        self._logger = getLogger()
         self._host = host
         self._sdk_port = sdk_port
         self._camera_port = camera_port
         self._restart_port = restart_port
         self._grpc_channel = grpc.insecure_channel(f'{self._host}:{self._sdk_port}')
 
+        if with_mobile_base:
+            try:
+                self.mobile_base = MobileBaseSDK(self._host)
+            except _InactiveRpcError:
+                self._logger.warning('Failed to connect to mobile base!')
+                return
```

Comment:
Raise an exception instead of a return? Maybe an IOError('Failed to connect to mobile base!')?